projects/core/components/tooltip/tooltip.component.ts

```diff
@@ -9,22 +9,17 @@ import {
 } from '@angular/core';
 import {TUI_IS_MOBILE, TuiDestroyService} from '@taiga-ui/cdk';
 import {TuiHintHoverDirective, TuiHintOptionsDirective} from '@taiga-ui/core/directives';
-import {MODE_PROVIDER} from '@taiga-ui/core/providers';
-import {TUI_MODE} from '@taiga-ui/core/tokens';
-import type {TuiBrightness} from '@taiga-ui/core/types';
-import {takeUntil} from 'rxjs';
 
 @Component({
     selector: 'tui-tooltip',
     templateUrl: './tooltip.template.html',
     styleUrls: ['./tooltip.style.less'],
     changeDetection: ChangeDetectionStrategy.OnPush,
-    providers: [TuiDestroyService, MODE_PROVIDER],
+    providers: [TuiDestroyService],
```

Comment:
nit: Is `TuiDestroyService` now redundant?

Reply:
Looks like it. I'll remove it later.